_data/banking.yml

```diff
@@ -195,6 +195,14 @@ websites:
     img: caisse_epargne.png
     lang: fr
 
+  - name: Caixa Directa
+    url: https://www.cgd.pt/
+    tfa: 
+      - sms
+    doc: https://www.cgd.pt/Ajuda/Seguranca/Pages/Autenticacao-Dupla-SMS-Token.aspx
+    img: caixa_directa.png
+    lang: pt
```

Comment:
```suggestion
```
`lang`-field must not be present, when one or more tfa-methods are available (it's only used for i18n-tweet-texts).

Reply:
Resolved by 
c6ab66f